common/service/mctp/mctp.c

```diff
@@ -51,14 +51,30 @@ static uint8_t set_thread_name(mctp *mctp_inst)
 	    mctp_inst->medium_type >= MCTP_MEDIUM_TYPE_MAX)
 		return MCTP_ERROR;
 
-	if (mctp_inst->medium_type == MCTP_MEDIUM_TYPE_SMBUS) {
+	switch (mctp_inst->medium_type) {
+	case MCTP_MEDIUM_TYPE_SMBUS:
+		printf("[%s] medium_type = SMBUS\n", __func__);
 		mctp_smbus_conf *smbus_conf = (mctp_smbus_conf *)&mctp_inst->medium_conf;
 		snprintf(mctp_inst->mctp_rx_task_name, sizeof(mctp_inst->mctp_rx_task_name),
 			 "mctprx_%02x_%02x_%02x", mctp_inst->medium_type, smbus_conf->bus,
 			 smbus_conf->addr);
 		snprintf(mctp_inst->mctp_tx_task_name, sizeof(mctp_inst->mctp_tx_task_name),
 			 "mctptx_%02x_%02x_%02x", mctp_inst->medium_type, smbus_conf->bus,
 			 smbus_conf->addr);
+		break;
+	case MCTP_MEDIUM_TYPE_I3C:
+		printf("[%s] medium_type = I3C\n", __func__);
```

Comment:
Change to use Zephyr logging service instead of printf.

Reply:
Complete modification